src/lib/components/compositions/checkbox/animated.svelte

```diff
@@ -1,16 +1,17 @@
 <script lang="ts">
   import { afterUpdate } from 'svelte';
 
-  import Error from '$components/primitives/message/error.svelte';
+  import Message from '$components/primitives/message/input-message.svelte';
   import Label from '$components/primitives/label/label.svelte';
 
   export let checkValidity: ((event: Event) => boolean) | null = null;
-  export let errorMessage = '';
+  export let message = '';
   export let firstInvalidInput: boolean;
   export let isRequired = false;
   export let isInvalid: boolean | null = null;
   export let key: string;
   export let onChange: (event: Event) => void;
+  export let showMessage: boolean | undefined = undefined;
```

Comment:
Adding an explicit control over showing the message or not. I could no longer determine whether to show it implicitly.